### ISSUE TITLE
Hello, I am following the style guide, but the question arose about how to keep the signals of the models as before and after saving and maintaining a separation of concerns

### ISSUE THREAD

**Issue by italoantonio0909** [HackSoftware/Styleguide-Example#21]

**Comment by RadoRado** [comment]
@italoantonio0909 My apologies for answering that late - we've just enabled notifications for this repository.

If you still have that question, I'd be happy to answer.

Signals are a great tool to achieve communication between apps that should not know about each other.

Signals are also great for cache invalidation.

What we do not recommend using Signals for is to "Create objects based on other objects being created/updated/etc", since the control flow of your apps becomes really hard to follow.

If you have additional questions, feel free to reopen the issue with them :v:
